src/components/OcSelect.vue

```diff
@@ -33,6 +42,11 @@ export default {
       default: null,
     },
   },
+  computed: {
+    setLabel() {
```

Comment:
i find this somehow misleading, sounds like a method. Maybe selectLabel?

Reply:
How about `inputLabel`? @kulmann any suggestions/opinions?

Reply:
yeah, `inputLabel` is fine